src/components/SearchPane/Controls/SearchPaneControls.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+
+import NavContext from 'NavContext/NavContext';
+import Button from 'Button/Button';
+import SearchContext from '../Context/SearchPaneContext';
+
+export const SearchPaneControls = () => {
+  const { toggleNav } = useContext(NavContext);
+  const { toggleSearch } = useContext(SearchContext);
+
+  const closeAll = () => {
+    toggleNav(false);
+    toggleSearch(false);
+  };
+
+  return (
+    <div className="search-pane__controls">
+      <Button
+        className="search-pane__close"
+        type="button"
+        onClick={closeAll}
+        icon="close"
+        styled={false}
+      >
+        Close <span className="u-visually-hidden">search</span>
+      </Button>
+      <Button
+        className="search-pane__back"
+        type="button"
```

Comment:
Prob worth setting `type = button` as default in Button component itself, it would save us having to set this prop each time the component is used

Reply:
Good point, will do.